ansible_collections/arista/avd/extensions/molecule/example-isis-ldp-ipvpn/molecule.yml

```
@@ -1,4 +1,15 @@
 ---
+ansible:
+  executor:
+    backend: ansible-playbook
+    args:
+      ansible_playbook:
+        - --inventory=../../../examples/isis-ldp-ipvpn/inventory.yml
+  env:
+    ANSIBLE_FORCE_COLOR: true
+  cfg:
```

this should fix it without needing the intermediate playbook: https://ansible.readthedocs.io/projects/molecule/ansible-native/#configuration-structure

```suggestion
  playbooks:
    converge: ../../../examples/isis-ldp-ipvpn/build.yml
  cfg:
```

---

implemented